src/site/content/en/identity/passkey-form-autofill/index.md

```diff
@@ -0,0 +1,191 @@
+tags:
+  - identity
+  - security
+---
+
+[Passkeys](https://developers.google.com/identity/passkeys) replace passwords
+and make user accounts on the web safer, simpler, easier to use. However, the
+transition from password-based to passkey-based authentication can complicate
+the user experience. Leveraging form autofill to suggest passkeys can help
+create a unified experience.
+
+## Why use form autofill to sign-in with a passkey?
+
+Using passkeys instead of passwords is a great way for websites to make their
+user accounts safer, simpler, easier to use. With a passkey, a
+user can sign in to a website or an app just by using fingerprint, face or
+device pin.
+
+In an ideal world, there would be no password users and the authentication flow
+could be as simple as a single sign-in button. When the user taps the button, an
+account selector dialog pops up, the user can pick an account, unlock the screen
+to verify and sign in.
+
+However, the transition from password-based authentication to passkey-based
+authentication can be challenging. As users switch to passkeys, there will still
+be those who use passwords and websites will need to accommodate both types of
+users. Users themselves should not be expected to remember on which sites
+they've switched to passkeys, so asking users to select which method to use up
+front would be poor UX.
+
+Passkeys are also a new technology. Explaining them and making sure users are
+comfortable using them can be a challenge for websites. We can rely on familiar
+user experiences for autofilling passwords to solve both problems.
+
+## Conditional UI
+
+To create an efficient user experience for both passkey and password users, you
+can optimize the authentication flow using the browser's form autofill
+functionality. This is called
+[conditional UI](https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI)
+and it's a part of [the WebAuthn standard](https://w3c.github.io/webauthn/). A
+website can include passkeys in autofill suggestions.
+
+1.  The user lands on the sign-in page.
+1.  The user selects the username input field.
+1.  An account selector dialog pops up.
+1.  The user selects an account, whether that account uses a password or a
+    passkey.
+1.  The user verifies with fingerprint, face or device PIN.
+1.  The user is signed in.
+
+As soon as the user taps on the username input field, an autofill suggestion
+dialog pops up which highlights the stored passkeys along with password autofill
+suggestions. The user can then choose an account and use the device screen lock
+to sign in.
+
+This way, users can sign in to your website with the existing form as if
+nothing has changed, but with
+[the added security benefit of passkeys](https://docs.google.com/document/d/1cPn7f-QJ_B10HYa_yN6LFsHKUJ051ydAPAg4N3hHBXM/edit#heading=h.gbh1vq1hvzqp)
```

Comment:
should this really link to a Google doc?

Reply:
It should be https://developers.google.com/identity/passkeys#security-considerations instead

Reply:
No, we'll replace it in a bit.